instagram/src/components/Authentication/Authenticate.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import LoginPage from '../LoginPage/LoginPage';
+
+
+const Authenticate = App => {
+    return class extends React.Component {
+        constructor(){
+            super();
+            this.state = {
+                loggedIn : undefined, 
+                loginUserText: '',
+                loginPasswordText: ''
+            }
+        }
+    
+        logOutTop = () => {
+            this.setState({
+                loggedIn : false
+            })
+        }
+
+        login(username, password) {
+            localStorage.setItem('user', username)
+            localStorage.setItem('password', password)
+        }
+
+        handleInputChange = event => {
+            this.setState({
+              [event.target.name]: event.target.value,
+            });
+        };
+
+        componentDidMount(){
```

Comment:
This took me a second to figure out how to login to your application.  Was the idea with hard coding this that you were experimenting with what actual authentication might look like?   I.e. you wanted to kick back to the login page whoever didn't have a valid username?